db/re/item_db_usable.yml

```diff
@@ -2272,6 +2272,93 @@ Body:
     Weight: 10
     Script: |
       /* upgradeui 9785; */
+  - Id: 9894
+    AegisName: aegis_9894
+    Name: "[Scroll] Adrenaline Lv.5 Scroll"
+    Type: Usable
+    Buy: 2
+    Weight: 10
+    NoUse:
+      Override: 100
+      Sitting: true
+    Trade:
+      Override: 100
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      .@type = getiteminfo(getequipid(EQI_HAND_R), ITEMINFO_VIEW);
+      if (.@type == W_1HAXE || .@type == W_2HAXE || .@type == W_MACE) {
+         skilleffect "BS_ADRENALINE",0;
+         sc_start SC_ADRENALINE,150000,5;
+      }
```

Comment:
I don't know what the correct implementation of this item should be, but I believe it should look like this one instead
```
  - Id: 12471
    AegisName: F_Adrenaline_Scroll
    Name: F Adrenaline Scroll
    Type: Delayconsume
    Buy: 2
    Weight: 10
    Script: |
      itemskill "BS_ADRENALINE",5;
```

Reply:
Thanks!